source/includes/opts/migrationName.rst

```diff
@@ -0,0 +1,18 @@
+.. reference/configuration.txt
+.. reference/mongosync.txt
+
+.. versionadded:: 1.7.0
+
+Sets a migration name for a sync operation. For example, you can set a
+migration name to identify the sync operations from multiple source
+clusters into one destination cluster.
+
+The ``migrationName`` string can contain up to 44 alphanumeric and
```

Comment:
It looks like migrationName also supports spaces, should we call this out in the description?

```suggestion
The ``migrationName`` string can contain up to 44 alphanumeric, space, and
```

Reply:
Cannot have spaces. Updated text.